nni/algorithms/compression/v2/pytorch/pruning/basic_scheduler.py

```diff
@@ -33,26 +34,33 @@ def __init__(self, pruner: Pruner, task_generator: TaskGenerator, finetuner: Cal
         evaluator
             Evaluate the pruned model and give a score.
             If evaluator is None, the best result refers to the latest result.
+        reset_weight
```

Comment:
reset_weight is True means every iteration uses original model weights, right?

Reply:
yes, finetune stage is placed at the beginning in each `step`, and the weight will be reset to the weight before finetuning then speedup.